vendor/grammars/blade/CHANGELOG.md

```diff
@@ -0,0 +1,99 @@
+# Change Log
```

Comment:
Because since the grammar's folder wasn't listed as a submodule, the entire repo's contents have now been tracked in your branch's version control history...